src/index.ts

```diff
@@ -240,8 +240,10 @@ const txBodies = async (req: Request, res: Response) => {
 
 const getStatus = async (req: Request, res:  Response) => {
   const mobilePlatformVersionPrefixes = ["android / ", "ios / ", "- /"];
+  const desktopPlatformVersionPrefixes = ["firefox / ", "chrome / "];
   const clientVersionHeader = 'yoroi-version';
-  const minMobileVersion = '2.2.2';
+  const minMobileVersion = '2.2.4';
+  const minDesktopVersion = '1.10.4';
```

Comment:
should this be `3.0.0` then?

Reply:
either `1.10.4` or `3.0.0` should work equally fine.

Reply:
okay.  anything else you'd like to see before merging-and-deploying ?